CONTRIBUTING.md

```diff
@@ -1,13 +1,14 @@
 I've found a bug!
 -----
-Awesome, now let's gather as much information as possible.
+Awesome, but hold up! First of all, make sure that someone else hasn't found this bug previously, if no releated issues found let's gather as much information as possible.
```

Comment:
...previously, and, if no related issues are found, let's...
OR
...previously; if no related issues are found, let's...